### DISCUSSION TITLE
Pre-selected values in filter web part

### DISCUSSION THREAD

**Discussion by MirasMNC** [microsoft-search/pnp-modern-search#4088] [Q&A] [answered]
Hi,

I would like to ask you if there is another way how to set preselected filters. I have found this method in documentation (https://microsoft-search.github.io/pnp-modern-search/usage/search-filters/#filter-deep-linking). I would prefer a way which is independent from URL adress because people can share different version of URL and filter will be lost. If there is a way how to set it directly on the web part within the settings e.g. some checkbox that it is default value. I am just asking, maybe somebody has had similar case.

Thank you

Mirek :)

**Comment by wobba** [comment, marked as answer]
Currently not an option. If someone develops this it would be a good addition.

**Reply by MirasMNC** [reply]
Thanks for fast response. :)
